code/game/objects/structures/reflector.dm

```diff
@@ -7,6 +7,8 @@
 	density = TRUE
 	layer = 3
 	var/finished = FALSE
+	var/obj/item/stack/sheet/buildstacktype
+	var/buildstackamount
```

Comment:
could you make these variables use `snack_case`. The spacing will help with readability.